tools/update-timezone.mjs

```diff
@@ -15,24 +14,12 @@ const availableVersions = readdirSync('icu-data/tzdata/icunew', { withFileTypes:
 .filter((dirent) => dirent.isDirectory())
 .map((dirent) => dirent.name);
 
-const currentVersion = process.versions.tz;
 const latestVersion = availableVersions.sort().at(-1);
 
-if (latestVersion === currentVersion) {
-  console.log(`Terminating early, tz version is latest @ ${currentVersion}`);
-  exit();
-}
-
 execSync('bzip2 -d deps/icu-small/source/data/in/icudt*.dat.bz2');
 fileNames.forEach((file) => {
   renameSync(`icu-data/tzdata/icunew/${latestVersion}/44/le/${file}`, `deps/icu-small/source/data/in/${file}`);
-  spawnSync(
-    'icupkg', [
-      '-a',
-      file,
-      'icudt*.dat',
-    ], { cwd: 'deps/icu-small/source/data/in/' }
-  );
+  execSync(`icupkg -a ${file} icudt*.dat`, { cwd: 'deps/icu-small/source/data/in/' });
```

Comment:
Wouldn't this be an issue if `file` contains "weird" characters?

Reply:
Maybe but that's never supposed to happen because `file` is always gonna be one of https://github.com/nodejs/node/blob/0d80e7344e53dd6c07297405aa785423e618a8cd/tools/update-timezone.mjs#L7-L12 and I think this list is unlikely to change.